django/contrib/admin/static/admin/css/changelists.css

```diff
@@ -148,12 +148,33 @@
     border-bottom: none;
 }
 
-#changelist-filter h3 {
+#changelist-filter details summary {
+    cursor: pointer;
     font-weight: 400;
     padding: 0 15px;
     margin-bottom: 10px;
 }
 
+#changelist-filter details summary > * {
+    display: inline;
+}
+
+#changelist-filter details > summary {
+    list-style-type: none;
+}
+
+#changelist-filter details > summary::-webkit-details-marker {
+    display: none;
+}
+
+#changelist-filter details > summary::before {
+    content: '▶️';
```

Comment:
With Linux|Chrome I see ▶️ like this:

![image](https://user-images.githubusercontent.com/16822952/163657145-8d0cb445-429d-41a4-a361-0bc0a7a82060.png)

Should I re-define or load any font?

Reply:
Can we use arrows instead of emojis? ↓ → :thinking:

Reply:
I'm not an UX expert, but they look nice and the same in all browsers:
- Firefox:
  ![image](https://user-images.githubusercontent.com/2865885/164675019-31737590-7509-4527-97f4-699d7b6fda62.png)
- Chrome:
  ![image](https://user-images.githubusercontent.com/2865885/164675166-9bd5c901-b938-4684-930a-d2d8f3f30fdc.png)

Maybe bolded and with `--link-hover-color` to make it clear that they are clickable:
```css
#changelist-filter details > summary::before {
    content: '→';
    font-weight: bold;
    color: var(--link-hover-color);
}

#changelist-filter details[open] > summary::before {
    content: '↓';
    font-weight: bold;
    color: var(--link-hover-color);
}
```

![image](https://user-images.githubusercontent.com/2865885/164675929-c9457022-a008-4fbd-97e1-809074b43b58.png)

Reply:
I really like it @felixxm! Styles updated.